web/src/actions/editActivity/activityList.js

```diff
@@ -3,7 +3,7 @@ import {
   APD_ACTIVITIES_CHANGE,
   EDIT_APD,
   REMOVE_APD_ITEM
-} from '../editApd/symbols';
+} from '../editApd';
```

Comment:
Based on @thetif comment:
> that's kind of the redux standard so that you can use them in the action and the reducer and avoid typos

we may want to keep it linked to the symbols. maybe we could revisit this in a tech debt ticket and decide if the trade offs are worth it

Reply:
ok.  replacing symbols in these files.